packages/flutter_web_plugins/lib/flutter_web_plugins.dart

```diff
@@ -15,8 +15,6 @@
 ///    describing how the `url_launcher` package was created using [flutter_web_plugins].
 library flutter_web_plugins;
 
-export 'src/navigation/js_url_strategy.dart';
-export 'src/navigation/url_strategy.dart';
-export 'src/navigation/utils.dart';
 export 'src/plugin_event_channel.dart';
 export 'src/plugin_registry.dart';
+export 'url_strategy.dart';
```

Comment:
I think removing these exports from here make this a breaking change.

In order to make this land more softly, maybe we can leave the old exports here (If the files are still somewhat compatible with the versions before this change), so people importing `package:flutter_web_plugins/flutter_web_plugins.dart` would get the old behavior, and then by **not exporting** `url_strategy.dart` from here, people who want the new behavior would just `import package:flutter_web_plugins/url_strategy.dart`.

What do you think? Am I overthinking the backwards compatibility of this?

Reply:
Good point.

Leaving this file as is leads to a breaking change though. Because `src/navigation/url_strategy.dart` doesn't export all the things it used to export. Few things have moved to `src/navigation_common/url_strategy.dart`.

How about leaving all existing exports, and adding `export 'src/navigation_common/url_strategy.dart';`?

Reply:
> How about leaving all existing exports, and adding export 'src/navigation_common/url_strategy.dart';?

@mdebbar yes, you're right, we need the common file too, so people have access to the classes that moved there.